packages/abi-typegen/src/templates/contract/dts.test.ts

```diff
@@ -74,4 +74,19 @@ describe('templates/dts', () => {
     const rendered = renderDtsTemplate({ abi });
     expect(rendered).toMatch(/export type MyEnumOutput = MyEnumInput;$/m);
   });
+
+  test('should not render same value for native identical enums', () => {
+    const project = getProjectResources(ForcProjectsEnum.ENUM_SIMPLE_NATIVE);
+    const { abiContents: rawContents } = project;
+
+    const abi = new Abi({
+      filepath: './my-contract-abi.json',
+      outputDir: 'stdout',
+      rawContents,
+      programType: ProgramTypeEnum.CONTRACT,
+    });
+
+    const rendered = renderDtsTemplate({ abi });
+    expect(rendered).toMatch(/export enum MyEnumOutput { Checked, Pending };$/m);
```

Comment:
Do you think this test is necessary?

If so, the assertion must be improved.

Reply:
I misread the assertion in my first review. As it is, the test makes sense.

However, we could reassign the enum to a const, which should have the same effect with fewer chars.

Just nitpicking, nothing to worry about.

```ts
export enum MyEnumInput { Checked = 'Checked', Pending = 'Pending' };
export const MyEnumOutput = MyEnumInput;
```